extensions/algebra/circuit/src/fp2_extension.rs

```diff
@@ -89,69 +90,67 @@ impl<F: PrimeField32> VmExecutionExtension<F> for Fp2Extension {
         &self,
         inventory: &mut ExecutorInventoryBuilder<F, Fp2ExtensionExecutor>,
     ) -> Result<(), ExecutorInventoryError> {
-        // TODO: add getter for pointer_max_bits
-        let pointer_max_bits = 29;
-        let range_checker_bus = inventory.range_checker().bus;
-        for (i, modulus) in self.supported_moduli.iter().enumerate() {
+        let pointer_max_bits = inventory.address_bits();
+        // TODO: somehow get the range checker bus from `ExecutorInventory`
+        let range_checker_bus = VariableRangeCheckerBus::new(1, 22);
```

Comment:
instead of 1, use `u16::MAX` or something and name this `dummy_range_bus` so it's clear it's not used